features/steps/steps.py

```diff
@@ -320,10 +320,57 @@ def step_impl(context):
 
 @when('the Player selects column 1')
 def step_impl(context):
-    context.child.expect('Player 1, its your turn.')
+    # Wait for prompt when cmd becomes ready.
```

Comment:
not sure but i guess this should work, someone confirm please @LouiBrown @jasminSK @JohniMIEP

Reply:
I didn´t write this part. I just copied it and changed it for the other columns.

Reply:
Should still work fine but unnecessary.